File: CoreModels/Journal.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace CoreModels
{
    public class Journal
    {
        
        public int ID { get; set; }
        public string Cpr { get; set; }
        public string Name { get; set; }
        public string MiddleName { get; set; }
        public string Lastname { get; set; }
        public string Info { get; set; }
        public Incident Incident { get; set; }
        public Reception Reception { get; set; }
        public DateTimeOffset CreationTime { get;  set; }
        public List<Logger> Loggers { get; set; }
        public string Status { get; set; }

    }
}

File: Admin/ClientApp/app/components/Vehicle/Journal.ts
export class Journal {
    id: number;
    cpr: string;
    name: string;
    middleName: string;
    lastname: string;
    info: string;
    incident: {
        id: number;
        address: string;
        number: string;
        postNr: string;
        city: string;
        description: string;
        status: string;
        journals: any[];
        preparednesses: any[];
        creationTime: Date;
        loggers: any[];
    };
    reception: {
        id: number;
        code: number;
        name: string;
        journals: any[];
    };
    creationTime: Date;
    loggers: any[];
    status: string;
}